lib/rules/indent.js

```diff
@@ -1039,6 +1039,11 @@ module.exports = {
 
             VariableDeclarator(node) {
                 if (node.init) {
+                    const equalOperator = sourceCode.getTokenBefore(node.init, astUtils.isNotOpeningParenToken);
+
+                    if (equalOperator) {
```

Comment:
I suppose that makes sense, but if that happens, then some invariant has been violated and it's likely that the rule will do the wrong thing anyway. So I think it's better to avoid unnecessary checks unless we know that they're necessary in a particular case.